be2-scala/src/main/scala/ch/epfl/pop/storage/DbActor.scala

```diff
@@ -857,10 +852,10 @@ object DbActor {
   final case class WriteRumor(rumor: Rumor) extends Event
 
   /** Requests the Db for rumors corresponding to keys {server public key:rumor id}
-    * @param desiredRumors
+    * @param desiredRumor
     *   Map of server public keys and list of desired rumor id for each
     */
-  final case class ReadRumors(desiredRumors: Map[PublicKey, List[Int]]) extends Event
+  final case class ReadRumors(desiredRumor: (PublicKey, Int)) extends Event
```

Comment:
If it returns only one rumor:
```suggestion
  final case class ReadRumor(desiredRumor: (PublicKey, Int)) extends Event
```